aait/backend/group-2B/Application/DTOs/InteractionDTOs/Validator/CommentDtoValidator.cs

```diff
@@ -0,0 +1,60 @@
+using FluentValidation;
+using SocialSync.Application.Contracts.Persistence;
+
+namespace SocialSync.Application.DTOs.InteractionDTOs.Validator;
+
+public class CommentDtoValidator : AbstractValidator<InteractionDto>
+{
+    private IUnitOfWork _unitOfWork;
+    public CommentDtoValidator(
+        IUnitOfWork unitOfWork
+    )
+    {
+        _unitOfWork = unitOfWork;
+
+        RuleFor(interaction => interaction.Body)
+            .NotEmpty()
+            .WithMessage("{ProprtyName} can not be empty")
+            .NotNull()
+            .MinimumLength(1)
```

Comment:
The MinLength validation is not required in this case because it is redundant when used with NotEmpty().

The NotEmpty() validation ensures that the Body property is not null and not an empty string. It covers the case where the length of the string must be at least one character. Therefore, adding MinimumLength(1) validation after NotEmpty() is unnecessary.